### DISCUSSION TITLE
Add energy cost calculator

### DISCUSSION THREAD

**Discussion by Langustensorbet** [CorentinTh/it-tools#650] [Ideas]
Hello, I have an idea that could enhance your IT tools. How about integrating an energy cost calculator? This calculator would allow users to input the wattage of a device, its daily usage duration, and the cost per kWh, and then provide them with estimates for daily, weekly, monthly, and yearly electricity consumption and expenses.
I believe this feature could be a valuable addition to your software. Thank you for considering my suggestion.

**Comment by sharevb** [comment]
Hi @Langustensorbet and @CorentinTh, implemented in #1372

**Comment by sharevb** [comment]
Hi, for people interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
